cli/pipelines/publish.ttl

```diff
@@ -19,21 +26,76 @@
     ] ;
   code:arguments
     (
+      <#cubeMetadata>
       <#PublishedLatestCube>
       <#ExpirePreviousVersions>
     )
 .
 
+<#cubeMetadata>
+  a :Pipeline, :ReadableObjectMode ;
+  :steps
+    [
+      :stepList (
+                  <#loadCubeMetadata>
+                  <#shaclValidate>
+                  <#flatten>
```

Comment:
The core change, separating the query for cube metadata from the observations themselves so that we can run SHACL validation on the metadata alone